### PR TITLE
bug: fix README.md #1516

### PR DESCRIPTION
**Pull request by sujay2306**
fixed the readme contributors bug issue using 
https://contrib.rocks/image?repo=wechaty/docusaurus
since it is dynamic and we need not to update contributors list manually 

@huan

### CODE REVIEW

**Comment by huan** [comment]
Please make sure the following requirements have been met for your PR before we can continue reviewing it:

2. The PR has passed all the CI tests and turned _green_. (you can execute all tests locally by running `npm test`)

File: README.md [hunk 1]
```diff
@@ -70,15 +70,18 @@ At the same time, we also meet up offline all over the world. Here are some acti
 - [Markdown Linting Rules Documents](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md)
 
 ## Writers
+<a href="https://github.com/wechaty/docusaurus/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=wechaty/docusaurus" />
+</a>
 
-[![contributor](https://sourcerer.io/fame/huan/wechaty/wechaty.js.org/images/0)](https://sourcerer.io/fame/huan/wechaty/wechaty.js.org/links/0)
+<!-- [![contributor](https://sourcerer.io/fame/huan/wechaty/wechaty.js.org/images/0)](https://sourcerer.io/fame/huan/wechaty/wechaty.js.org/links/0)
```
Comment (huan) [inline README.md]: If this part is not working anymore, I think you can remove them after confirm the new code works well.

Comment (sujay2306) [reply README.md]: @huan i have removed this now

**Comment by huan** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> You've invited Sujay to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.  

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message. (See also: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16))

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/jekyll/tree/main/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add @lijiarui wechat: _ruirui_0914_ and send her this pr link. She will invite you into Wechaty Contributor Room

Cheers!

**Comment by huan** [comment]
Please make sure the following requirements have been met for your PR before we can continue reviewing it:

2. The PR has passed all the CI tests and turns _green_. (you can execute all tests locally by running `npm test`)

**Comment by sujay2306** [comment]
@huan i have fixed the issue now npm test cases have passed

**Comment by huan** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message. (See also: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16))

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/jekyll/tree/main/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add @lijiarui wechat: _ruirui_0914_ and send her this pr link. She will invite you into Wechaty Contributor Room

Cheers!

**Pull request merged**